include/net/socket.h

```diff
@@ -275,6 +280,21 @@ struct addrinfo {
 };
 
 #include <net/socket_offload.h>
+
+static inline int inet_pton(sa_family_t family, const char *src, void *dst)
```

Comment:
As we discussed offline, the other alternative would have been to add this into the socket_offload_ops.h table, but this looks like the simpler approach.

Reply:
Yes, if we decide to truly offload this operation in the future (by calling into a SimpleLink-based API), we can revisit this and add that extra function pointer.